packages/adblocker/src/events.ts

```diff
@@ -52,6 +52,9 @@ function unregisterCallback<EventNames>(
     if (indexOfCallback !== -1) {
       listenersForEvent.splice(indexOfCallback, 1);
     }
+    if (listenersForEvent.length === 0) {
```

Comment:
is this needed?

Reply:
This was added to optimise `hasListeners` but it's no longer used now.